org.nongnu.gsequencer.json

```diff
@@ -0,0 +1,241 @@
+{
+    "app-id": "org.nongnu.gsequencer",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.32",
+    "sdk": "org.gnome.Sdk",
+    "command": "gsequencer",
+    "rename-desktop-file": "gsequencer.desktop",
+    "rename-appdata-file": "gsequencer.appdata.xml",
+    "rename-icon": "gsequencer",
+    "finish-args": [
+	/* X11 + XShm access */
+	"--share=ipc",
+	"--socket=x11",
+	/* Wayland */
+	"--socket=wayland",
```

Comment:
if this is using gtk2 does wayland actually work?

Reply:
I think yes as starting wayland from gdm I was able to launch the application.

Reply:
X applications still work via XWayland. Its still an X application.

Reply:
So do I need to remove?

Reply:
It does nothing so yes.